package.json

```diff
@@ -79,6 +79,8 @@
     }
   },
   "dependencies": {
+    "cli-table3": "^0.6.2",
```

Comment:
I'd like to get @OrKoN's comments on using this specific library (and overall, using a 3rd party library for this feature). WDYT?

Reply:
The code looks good to me but I was wondering if it's better to integrate with existing reporters (e.g., such as mocha/jest reporters) instead of implementing our own reporter?

Reply:
Using an existing library can reduce the maintenance cost. 👍 Which library do you suggest? @OrKoN

Reply:
you know, let's start with a custom implementation for now! Later we can look into integrations with existing reporters out there. Note that being a full-fledged test runner is not a priority/goal for this library.